gradle.properties

```
@@ -11,7 +11,7 @@ artifact_basename=appliedenergistics2
 minecraft_release=1.16
 minecraft_version=1.16.1
 mcp_mappings=20200723-1.16.1
-forge_version=32.0.93
+forge_version=32.0.107
```

```suggestion
forge_version=32.0.108
```

Their breaking change in 107 appears to be a bit premature and causes the event to fire way too often.